openlibrary/plugins/openlibrary/home.py

```diff
@@ -313,24 +221,21 @@ def get_authors(doc):
     work = book.works and book.works[0]
     if work:
         d.authors = get_authors(work)
+        cover = work.get_cover()
     else:
         d.authors = get_authors(book)
+        cover = book.get_cover()
 
-    cover = book.get_cover()
     if cover:
         d.cover_url = cover.url("M")
 
     ia_id = book.get("ocaid")
     if ia_id:
         d.ocaid = ia_id
         collections = ia.get_meta_xml(ia_id).get("collection", [])
-        if 'printdisabled' in collections or 'lendinglibrary' in collections:
-            d.daisy_url = book.url("/daisy")
 
-        if 'lendinglibrary' in collections:
+        if 'lendinglibrary' in collections or 'inlibrary' in collections:
```

Comment:
`lendinglibrary`is deprecated. You can remove any references to it. Lendable books are only in `inlibrary`.